.github/dependabot.yml

```diff
@@ -1,10 +1,21 @@
 version: 2
 updates:
-  - package-ecosystem: github-actions
-    directory: /
+  - package-ecosystem: "npm"
+    directory: "/"
     schedule:
-      interval: weekly
-  - package-ecosystem: npm
-    directory: /
+      interval: "weekly"
+    commit-message:
+      prefix: "fix"
```

Comment:
do you want this to include the `:` and maybe a space?

Reply:
I think I don't need it. Check the docs here: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message

In the example they give, they have `prefix-development: "pip dev"` and later give the commit message as `pip dev: bump requests from 1.0.0 to 1.0.1`.

Reply:
docs work for me!